gotestcover.go

```diff
@@ -235,7 +240,11 @@ func writeCoverProfile(cov []byte) error {
 }
 
 func runGoCommand(args ...string) ([]byte, error) {
-	cmd := exec.Command("go", args...)
+	goCmd := "go"
+	if flagGoAppEngine {
+		goCmd = "goapp"
```

Comment:
Can you confirm `goapp test` support the same flags as `go test`?
I can't test it right now.